app/src/main/java/org/p2p/wallet/moonpay/model/MoonpayBuyQuote.kt

```diff
@@ -0,0 +1,11 @@
+package org.p2p.wallet.moonpay.model
+
+import org.p2p.wallet.home.model.Token
+import java.math.BigDecimal
+
+data class MoonpayBuyQuote(
+    val currency: String,
```

Comment:
why won't you use Currency class here

Reply:
No need to store all Currency class data, min and max amount. Considering that we have minAmount inside `MoonpayBuyQuote`